### ISSUE TITLE
After update to js-controller 3.3.1 several wrong types in log

### ISSUE THREAD

**Issue by Grothesk242** [AlCalzone/ioBroker.tradfri#578]
```
nodejs -v && npm -v && iobroker -v
v12.22.1
6.14.12
3.3.1
tradfri 2.7.1
```
All other bulbs stay silent, but these 5 bulbs are set up to 'always power on' when power is restored. 
```
2021-04-30 14:09:40.853  - warn: tradfri.0 (26233) State value to set for "tradfri.0.L-65599.lightbulb.whenPowerRestored" has wrong type "undefined" but has to be "number"
2021-04-30 14:09:40.905  - warn: tradfri.0 (26233) State value to set for "tradfri.0.L-65597.lightbulb.whenPowerRestored" has wrong type "undefined" but has to be "number"
2021-04-30 14:09:40.926  - warn: tradfri.0 (26233) State value to set for "tradfri.0.L-65601.lightbulb.whenPowerRestored" has wrong type "undefined" but has to be "number"
2021-04-30 14:09:40.932  - warn: tradfri.0 (26233) State value to set for "tradfri.0.L-65598.lightbulb.whenPowerRestored" has wrong type "undefined" but has to be "number"
2021-04-30 14:09:40.951  - warn: tradfri.0 (26233) State value to set for "tradfri.0.L-65600.lightbulb.whenPowerRestored" has wrong type "undefined" but has to be "number"
```

**Comment by Grothesk242** [comment]
Having installed 2.7.2. the log still shows some issues:

```
2021-05-14 20:03:05.916  - info: tradfri.0 (30855) starting. Version 2.7.2 in /opt/iobroker/node_modules/iobroker.tradfri, node: v16.1.0, js-controller: 3.3.10
2021-05-14 20:03:06.026  - info: tradfri.0 (30855) Discovering the gateway automatically...
2021-05-14 20:03:06.102  - info: tradfri.0 (30855) Found gateway gw-abcdefghijk at 192.168.178.50
2021-05-14 20:03:07.194  - info: tradfri.0 (30855) got all devices: [65573,65590,65579,65551,65591,65594,65595,65559,65580,65585,65599,65560,65589,65558,65557,65564,65576,65572,65601,65600,65598,65597,65587,65566,65563,65570,65552,65571,65542,65577,65545,65584,65546,65583,65565,65575,65581,65582]
2021-05-14 20:03:07.771  - info: tradfri.0 (30855) received all devices
2021-05-14 20:03:07.787  - info: tradfri.0 (30855) got all groups: [131079,131090,131097,131084,131085,131076,131091,131089,131087,131094,131082]
2021-05-14 20:03:07.837  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131076.color"
2021-05-14 20:03:07.838  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131076.colorTemperature"
2021-05-14 20:03:07.838  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131076.hue"
2021-05-14 20:03:07.839  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131076.position"
2021-05-14 20:03:07.839  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131076.saturation"
2021-05-14 20:03:07.839  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131076.transitionDuration"
2021-05-14 20:03:07.880  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131079.color"
2021-05-14 20:03:07.881  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131079.colorTemperature"
2021-05-14 20:03:07.881  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131079.hue"
2021-05-14 20:03:07.882  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131079.position"
2021-05-14 20:03:07.882  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131079.saturation"
2021-05-14 20:03:07.883  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131079.transitionDuration"
2021-05-14 20:03:07.919  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131082.color"
2021-05-14 20:03:07.920  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131082.colorTemperature"
2021-05-14 20:03:07.920  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131082.hue"
2021-05-14 20:03:07.921  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131082.position"
2021-05-14 20:03:07.921  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131082.saturation"
2021-05-14 20:03:07.921  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131082.transitionDuration"
2021-05-14 20:03:07.957  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131084.color"
2021-05-14 20:03:07.958  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131084.colorTemperature"
2021-05-14 20:03:07.958  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131084.hue"
2021-05-14 20:03:07.959  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131084.position"
2021-05-14 20:03:07.959  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131084.saturation"
2021-05-14 20:03:07.959  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131084.transitionDuration"
2021-05-14 20:03:07.997  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131085.color"
2021-05-14 20:03:07.997  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131085.colorTemperature"
2021-05-14 20:03:07.998  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131085.hue"
2021-05-14 20:03:07.998  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131085.position"
2021-05-14 20:03:07.998  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131085.saturation"
2021-05-14 20:03:07.999  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131085.transitionDuration"
2021-05-14 20:03:08.034  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131087.color"
2021-05-14 20:03:08.035  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131087.colorTemperature"
2021-05-14 20:03:08.035  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131087.hue"
2021-05-14 20:03:08.035  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131087.position"
2021-05-14 20:03:08.036  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131087.saturation"
2021-05-14 20:03:08.036  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131087.transitionDuration"
2021-05-14 20:03:08.072  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131089.color"
2021-05-14 20:03:08.072  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131089.colorTemperature"
2021-05-14 20:03:08.073  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131089.hue"
2021-05-14 20:03:08.073  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131089.position"
2021-05-14 20:03:08.073  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131089.saturation"
2021-05-14 20:03:08.074  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131089.transitionDuration"
2021-05-14 20:03:08.109  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131090.color"
2021-05-14 20:03:08.109  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131090.colorTemperature"
2021-05-14 20:03:08.109  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131090.hue"
2021-05-14 20:03:08.110  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131090.position"
2021-05-14 20:03:08.110  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131090.saturation"
2021-05-14 20:03:08.110  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131090.transitionDuration"
2021-05-14 20:03:08.146  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131091.color"
2021-05-14 20:03:08.147  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131091.colorTemperature"
2021-05-14 20:03:08.147  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131091.hue"
2021-05-14 20:03:08.148  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131091.position"
2021-05-14 20:03:08.148  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131091.saturation"
2021-05-14 20:03:08.148  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131091.transitionDuration"
2021-05-14 20:03:08.189  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131094.color"
2021-05-14 20:03:08.190  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131094.colorTemperature"
2021-05-14 20:03:08.190  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131094.hue"
2021-05-14 20:03:08.190  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131094.position"
2021-05-14 20:03:08.191  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131094.saturation"
2021-05-14 20:03:08.191  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131094.transitionDuration"
2021-05-14 20:03:08.194  - info: tradfri.0 (30855) State value to set for "tradfri.0.G-131091.stopBlinds" has to be type "boolean" but received type "function"
2021-05-14 20:03:08.207  - info: tradfri.0 (30855) State value to set for "tradfri.0.G-131094.stopBlinds" has to be type "boolean" but received type "function"
2021-05-14 20:03:08.224  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131097.color"
2021-05-14 20:03:08.224  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131097.colorTemperature"
2021-05-14 20:03:08.225  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131097.hue"
2021-05-14 20:03:08.225  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131097.position"
2021-05-14 20:03:08.225  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131097.saturation"
2021-05-14 20:03:08.225  - info: tradfri.0 (30855) undefined is not a valid state value for id "tradfri.0.G-131097.transitionDuration"
2021-05-14 20:03:08.253  - info: tradfri.0 (30855) State value to set for "tradfri.0.G-131097.stopBlinds" has to be type "boolean" but received type "function"
2021-05-14 20:03:08.257  - info: tradfri.0 (30855) got all scenes in group 131076: []
2021-05-14 20:03:08.291  - info: tradfri.0 (30855) got all scenes in group 131079: []
2021-05-14 20:03:08.320  - info: tradfri.0 (30855) got all scenes in group 131082: []
2021-05-14 20:03:08.351  - info: tradfri.0 (30855) got all scenes in group 131084: []
2021-05-14 20:03:08.384  - info: tradfri.0 (30855) got all scenes in group 131085: []
2021-05-14 20:03:08.417  - info: tradfri.0 (30855) got all scenes in group 131087: []
2021-05-14 20:03:08.453  - info: tradfri.0 (30855) got all scenes in group 131089: []
2021-05-14 20:03:08.485  - info: tradfri.0 (30855) got all scenes in group 131090: []
2021-05-14 20:03:08.518  - info: tradfri.0 (30855) got all scenes in group 131091: []
2021-05-14 20:03:08.549  - info: tradfri.0 (30855) got all scenes in group 131094: [196671]
2021-05-14 20:03:08.579  - info: tradfri.0 (30855) got all scenes in group 131097: []
2021-05-14 20:03:08.621  - info: tradfri.0 (30855) received all groups and scenes
```

**Comment by AlCalzone** [comment]
Yeah I missed one location. Should be fixed in 2.7.3 which I'm planning to release this weekend.

**Comment by AlCalzone** [comment]
2.7.3 released
